thunder/torch/__init__.py

```diff
@@ -2802,7 +2802,7 @@ def scatter(
         return clang.scatter(a, index, value, dim)
 
 
-@torchsymbol(torch.Tensor.scatter_, tags=(prims.OpTags.IN_PLACE,))
+@torchsymbol(torch.Tensor.scatter_, is_method=True, tags=(prims.OpTags.IN_PLACE,))
```

Comment:
Do we need the `is_method` here? My impression was that it's implicit when the symbol is on torch.Tensor...

Reply:
Yes, we do. Without it the linked-to line in NeVA will die in thunder with `AttributeError: Unknown attribute scatter_. Did you mean: '_return_value'?`

Reply:
Right, sorry. :)

Reply:
Oops, missed that. Thank you, @tfogal !